examples/custom-state-storage/mongo-state-storage.js

```diff
@@ -0,0 +1,62 @@
+const mongodb = require('mongodb');
+const MongoClient = mongodb.MongoClient;
+const Bluebird = require('bluebird');
+
+Bluebird.promisifyAll(MongoClient);
+
+class MongoDbStore {
+    constructor() {
+        this.url = 'mongodb://localhost/test';
+    }
+
+    connect() {
```

Comment:
Since this is an example, it would be best to keep it as simple as possible.  To that goal, I think it would be better to just `MongoClient.connect` in both `load` and `save.